### ISSUE TITLE
Add a way of computing logits easily during inference

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/pytorch-metric-learning#176]
Right now you can only compute the loss, but there are cases where people want to do classification using the logits (see https://github.com/KevinMusgrave/pytorch-metric-learning/issues/175)
